registries/adapters/openshift_adapter.go

```diff
@@ -43,7 +42,7 @@ type OpenShiftImage struct {
 
 // RegistryName - Retrieve the registry name
 func (r OpenShiftAdapter) RegistryName() string {
-	return openShiftName
+	return strings.TrimPrefix(r.Config.URL.String(), "https://")
```

Comment:
what if I use `http://<url>` as my URL? Would prefer to see it validate the string first